pkgs/top-level/qt5-packages.nix

```diff
@@ -227,6 +227,8 @@ in (kdeFrameworks // plasmaMobileGear // plasma5 // plasma5.thirdParty // kdeGea
 
   telepathy = callPackage ../development/libraries/telepathy/qt { };
 
+  tokodon = callPackage ../applications/misc/tokodon { };
```

Comment:
Oh sorry I guess I was wrong with the file `qt5-packages.nix`. It would be better to move it to `all-packages.nix` with a line like this:
```
zokodon = libsForQt5.callPackage ../applications/misc/tokodon { };
```
Sorry for my mistake